packages/fluentui/react-bindings/src/styles/types.ts

```diff
@@ -52,18 +57,6 @@ export type ComponentDesignProp = {
   maxHeight?: ICSSInJSStyle['maxHeight'];
 };
 
-export type RendererParam = {
-  theme: { direction: 'ltr' | 'rtl' };
-  disableAnimations: boolean;
-  displayName: string;
-  sanitizeCss: boolean;
-};
-
-export type RendererRenderRule = (rule: () => ICSSInJSStyle, param: RendererParam) => string;
-export type Renderer = Omit<FelaRenderer, 'renderRule'> & {
-  renderRule: RendererRenderRule;
-};
```

Comment:
Move out to `@fluentui/styles`, not sure that it's a right place, but:
- it's definitely not a part of `react-bindings`
- it should be available for both renderer packages

Reply:
Otherwise we can have a circular dependency:

```
@fluentui/react-northstar-emotion-renderer: error TS6202: Project references may not form a circular graph. Cycle detected: /mnt/work/1/s/packages/fluentui/react-northstar-emotion-renderer/tsconfig.json
@fluentui/react-northstar-emotion-renderer: /mnt/work/1/s/packages/fluentui/react-bindings/tsconfig.json
@fluentui/react-northstar-emotion-renderer: /mnt/work/1/s/packages/fluentui/react-northstar-fela-renderer/tsconfig.json
```

Reply:
https://github.com/microsoft/fluentui/pull/13602/files#r440234899 ?